src/main/java/core/basesyntax/controller/ControllerDaoImpl.java

```diff
@@ -0,0 +1,36 @@
+package core.basesyntax.controller;
+
+import core.basesyntax.dao.StorageDao;
+import core.basesyntax.dao.StorageDaoImpl;
+import core.basesyntax.model.Product;
+import java.util.List;
+
+public class ControllerDaoImpl implements ControllerDao<Product> {
+    private StorageDao<Product> storageDao;
+
+    public ControllerDaoImpl() {
+        this.storageDao = new StorageDaoImpl();
+    }
+
+    @Override
+    public boolean put(Product value) {
+        if (value == null) {
+            return false;
+        }
+        return storageDao.put(value);
+    }
+
+    @Override
+    public Product get(int index) {
```

Comment:
get() means you retrieving data from some place, in your case you remove them, let's change naming